src/Raven.Studio/typescript/components/pages/database/settings/documentRevisions/EditRevision.tsx

```diff
@@ -142,7 +145,17 @@ export default function EditRevision(props: EditRevisionProps) {
                         Purge revisions on document delete
                     </FormSwitch>
                     <FormSwitch control={control} name="isMinimumRevisionsToKeepEnabled">
-                        Limit # of revisions to keep
+                        <span>
+                            Set the number of revisions to keep
+                            <br />
+                            <span className="text-muted">
+                                (
+                                {isAgeRetentionActive
+                                    ? "When age-based retention is configured, this is the minimum number of revisions to keep"
```

Comment:
```suggestion
                                    ? "Age-based retention is configured, this is the minimum number of revisions to keep"
```

As i read it, `when` could mean that i may not have `age-based retention` configured yet, which could be slightly misleading in my opinion.

Reply:
We can see that this tooltip is conditional (`isAgeRetentionActive`), so the user has age-based retention configured.

Reply:
modified to:
```
{isAgeRetentionActive
    ? "The minimum number of revisions to keep (age-based retention enabled)"
    : "The maximum number of revisions to keep"}
 ```